eng/tools/oav-runner/tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist",
+    "rootDir": ".",
+    "noImplicitReturns": true,
```

```suggestion
    "noImplicitReturns": true,
```

Is this necessary?  It's not set in TSV.